src/Console/PhpToPo.php

```diff
@@ -15,6 +17,16 @@ class PhpToPo extends \Asika\SimpleConsole\Console
 	private $normBaseMsgIds = [];
 	const NORM_REGEXP = "|[\\\]|";
 
+	/** @var App */
+	private $app;
+
+	public function __construct(App $app, array $argv = null)
```

Comment:
Have you tested both this console command?

Reply:
yep, it does work because Dice is creating the console classes and automatically constructs the `App` class :-)